client/src/components/AbsoluteDateTime.tsx

```diff
@@ -0,0 +1,31 @@
+import moment from 'moment'
+import { HTMLProps } from 'react'
+import { Tooltip } from 'antd'
+import { useFormatter } from '../hooks/useFormatter'
+import { useServerMoment } from '../hooks/useServerTimeOffset'
+
+interface AbsoluteDateTimeProps extends HTMLProps<HTMLTimeElement> {
+  created: Date
+  updated: Date
+}
+
+const AbsoluteDateTime = (props: AbsoluteDateTimeProps) => {
```

Comment:
The component name is a bit misleading. Something like `EntityModificationDates` maybe?
Everything else looks good! Nice job :)

Reply:
I think so, too

Reply:
renamed to ModifucationTime